### DISCUSSION TITLE
mp3 backward forward

### DISCUSSION THREAD

**Discussion by Rhanem** [schreibfaul1/ESP32-audioI2S#189] [General]
Hi, it possible can fast forward and backward **_within the same SD mp3 song_**, especially when the song is long time?

**Comment by schreibfaul1** [comment]
setAudioPlayPosition(uint16_t sec) or setTimeOffset(int sec)

**Comment by Rhanem** [comment]
I have installed the old library that does not have the function setAudioPlayPosition nor the setTimeOffset .. I am going to remove the old and reinstall the new and see.. thamk you

**Comment by Rhanem** [comment]
thanks it was a good help, I have noticed that setAudioPlayPosition(uint16_t sec) always goes to a fixed position, for example if I choose 70 seconds even though I am at 100 or less or many more it returns me to the first 70 every time I press the button while the setTimeOffset(int sec) makes me jump for example the first time to 70 the second to 140 the third to 210 and so on ... that's what I wanted, thank you.
